internal/printer/print-to-js.go

```diff
@@ -479,6 +485,43 @@ func render1(p *printer, n *Node, opts RenderOptions) {
 	}
 }
 
+func printComponentImports(p *printer, doc *Node, source []byte) {
```

Comment:
Generally like to put these `printer` utils in `printer/printer.go` as a method on the `*printer` itself!

Reply:
Done!